src/layouts/layout.jsx

```diff
@@ -77,16 +111,12 @@ function Layout({ children }) {
                     height={56}
                   />
                 </Link>
-                <NavMenu />
+                {handleMenuNavShows()}
```

Comment:
I think to can do it more simpler:
```suggestion
                {!isMobile && <NavMenu />}
```

Reply:
done